src/generator/from_proto.rs

```diff
@@ -274,7 +281,6 @@ impl Context {
     /// Processes proto service definitions and their methods.
     fn append_query_service(mut self, services: &Vec<ServiceDescriptorProto>) -> Self {
         if services.is_empty() {
-            self.config = Config::default();
```

Comment:
I think this is required.. We don't support mutation as of now and query is resolved at the end, so if there is no query defined then it should generate empty config

Reply:
Yes but I don't think it should not be done here since this causes a problem when there are separate files for messages and services. I have added a method `remove_unused_types` in `Config` which will find all the unused types and remove them